tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -1362,9 +1361,6 @@ class DxcCompiler : public IDxcCompiler3,
     compiler.getLangOpts().HLSLProfile =
           compiler.getCodeGenOpts().HLSLProfile = Opts.TargetProfile;
 
-    compiler.getCodeGenOpts().HLSLEnablePartialLifetimeMarkers =
```

Comment:
This was here because this expression encodes the fact that "partial-lifetime-markers" is default off, and we did not want to repeat that logic in many places, so store the result in a flag on CodeGenOpts. Now that default-off is stored in `hlsl::options::TOGGLE_PARTIAL_LIFETIME_MARKERS`, we can just check it every time.